Example/Messaging/Tests/FIRMessagingRemoteNotificationsProxyTest.m

```diff
@@ -181,26 +176,26 @@ - (void)tearDown {
 
 - (void)testSwizzlingNonAppDelegate {
   RandomObject *invalidAppDelegate = [[RandomObject alloc] init];
-  [OCMStub([self.mockSharedApplication delegate]) andReturn:invalidAppDelegate];
+  [[GULAppDelegateSwizzler sharedApplication] setDelegate:(id<UIApplicationDelegate>)invalidAppDelegate];
```

Comment:
Any concerns here about using a singleton for these tests?

Reply:
Maybe, @maksymmalyhin can answer for sure. However, this fixes the tests being completely broken on iOS 8. See #3094 and #3093

Reply:
Specifically, see https://github.com/firebase/firebase-ios-sdk/blob/3ab4e388648c6f441717e8cbc68155dd5df7e335/GoogleUtilities/Example/Tests/Swizzler/GULAppDelegateSwizzlerTest.m#L645

Reply:
SGTM, let's keep an eye out for flakes just in case.